roles/cloudfront/templates/distribution.json

```diff
@@ -0,0 +1,108 @@
+{
+  "CallerReference": "{{ item.name | mandatory }}",
+  "Aliases": {
+    "Quantity": {{ item.alias_domains|count }},
+    "Items": {{ item.alias_domains }}
+  },
+  "DefaultRootObject": "{{ item.root_object|default('*') }}",
+  "PriceClass": "{{ item.price_class|default(PriceClass_100) }}",
+  "Origins": {
+    "Quantity": 1,
+    "Items": [
+      {
+        "Id": "{{ item.origin }}-id",
+        "DomainName": "{{ item.origin }}",
+        "OriginPath": "",
+        "CustomHeaders": {
+          "Quantity": 0
+        },
+        "CustomOriginConfig": {
+          "HTTPPort": 80,
+          "HTTPSPort": 443,
+          "OriginProtocolPolicy": "match-viewer",
+          "OriginSslProtocols": {
+            "Quantity": 4,
+            "Items": [
+              "SSLv3",
+              "TLSv1",
```

Comment:
We're not sure what the clients are going to be using seeing as this is open to the great unwashed eventually